libraries/botframework-connector/src/teams/models/teamsMappers.ts

```diff
@@ -3,6 +3,8 @@
  * Licensed under the MIT License.
  */
 
+export { ErrorResponse, ErrorModel, InnerHttpError } from '../../connectorApi/models/mappers';
```

Comment:
These exports, along with all the newly added exports are required for the ms-rest-mapper to correctly map the request/response body within the operation spec.